homeassistant/components/imap/const.py

```diff
@@ -11,6 +11,7 @@
 CONF_MAX_MESSAGE_SIZE = "max_message_size"
 CONF_CUSTOM_EVENT_DATA_TEMPLATE: Final = "custom_event_data_template"
 CONF_SSL_CIPHER_LIST: Final = "ssl_cipher_list"
+CONF_ENFORCE_POLLING: Final = "enforce_polling"
```

Comment:
Should this be called `force_polling`?

Reply:
Another option would be to `disable_push` and have them uncheck a box for `Push updates`

Reply:
Or `enable_push` which defaults to `True` if not defined in the options and have them uncheck a box for `Push updates`

Reply:
If have implemented an `enable_push` option that is enabled by default and can be turned off.